src/frontend/komponenter/Behandling/Inngangsvilkår/Sivilstand/SivilstandVurdering.tsx

```diff
@@ -10,7 +10,13 @@ import {
     skalViseLagreKnappSivilstand,
 } from '../../Vurdering/VurderingUtil';
 import Unntak from '../../Vurdering/Unntak';
-import { SivilstandType } from '../../../../typer/personopplysninger';
+import {
+    erEnkeEllerEnkemann,
+    erEnkeEllerGjenlevendePartner,
+    erIkkeUformeltGiftEllerSeparert,
+    erKravForSivilstandOppfylt,
+    erSkiltEllerUgift,
```

Comment:
Samme som for erEnkeEllerEnkemann - ser ikke denne i sivilstandHelper 🤔

Reply:
Fjernet i ny commit